app/components/FileUploader.tsx

```
@@ -1,53 +1,55 @@
+        onFileSelect?.(null);
+    };
 
     return (
         <div className="w-full gradient-border">
             <div {...getRootProps()}>
                 <input {...getInputProps()} />
 
                 <div className="space-y-4 cursor-pointer">
-                    {file ? (
+                    {selectedFile ? (
                         <div className="uploader-selected-file" onClick={(e) => e.stopPropagation()}>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)